geocity/apps/submissions/shortcuts.py

```diff
@@ -2,20 +2,33 @@
 
 from . import models
 from .exceptions import BadSubmissionStatus
+from .payments.models import ServicesFees
 
 
-def get_submission_for_user_or_404(user, submission, statuses=None):
+def get_submission_for_user_or_404(user, submission_id, statuses=None):
     """
-    Return the submission with `submission` or raise an Http404 if there is no such submission. The
+    Return the submission with `submission_id` or raise an Http404 if there is no such submission. The
     submission must either belong to the given user, or the given user should be in the same administrative entity.
     If `statuses` is set and a submission is found but its status doesn't match any value in `statuses`,
     `BadSubmissionStatus` will be raised.
     """
     permit_request = get_object_or_404(
-        models.Submission.objects.filter_for_user(user), pk=submission
+        models.Submission.objects.filter_for_user(user), pk=submission_id
```

Comment:
What is the rationale for this change ? This function beeing used in many places, are you sure it does't brake things ?

Reply:
This is only a cosmetic change.